src/Pim/index.tsx

```diff
@@ -220,6 +220,11 @@ const CollectionRoutes = withStyles(styles)(withRouter(
   }
 ));
 
+export const PimContext = React.createContext({
+  onItemSave: (_item: PimType, _journalUid: string, _originalEvent?: PimType, _goBack = true) => { return },
+  collectionsTaskList: [] as EteSync.CollectionInfo[],
+});
+
```

Comment:
Context does not seem like the correct tool for this job... Why can't it be passed as props? I must be missing something, but I don't understand what you are trying to do.
Also, have tasks specific code in an otherwise generic file is a code smell.